package.json

```diff
@@ -68,7 +69,7 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/rjrodger/seneca.git"
+    "url": "git+ssh://git@github.com:Beats/seneca.git"
```

Comment:
will need to change this.  Please update to https://github.com/senecajs/seneca.git